lib/flipper/api/error_response.rb

```diff
@@ -23,6 +23,7 @@ def as_json
       ERRORS = {
         feature_not_found: Error.new(1, "Feature not found.", "", 404),
         group_not_registered: Error.new(2, "Group not registered.", "", 404),
+        percentage_invalid: Error.new(3, "Percentage must be a positive number less than or equal to 100.", "", 400),
```

Comment:
Missed this in my review. Should this be 422? 400 is bad request and that isn't accurate as if it gets here it was a well formed request, it just wasn't valid. I think 422 is what rails typically uses for this type of error. Thoughts?